### ISSUE TITLE
\\n and \n not distinguished in translation interface

### ISSUE THREAD

**Issue by MichaelChirico** [MichaelChirico/potools#95]
As seen in `translate_package("unusual_msg", "es"):

```
File: /home/michael/github/potools/tests/testthat/test_packages/unusual_msg/R/foo.R
Call: message("\\n vs \n is OK")
Message: \n vs \n is OK
         ^^    ^^      
How would you translate this message into Spanish?
```

**Comment by MichaelChirico** [comment]
Ditto for quotes:

```
File: /home/michael/github/potools/tests/testthat/test_packages/unusual_msg/R/foo.R
Call: message("strings with escaped \"quotes\" are OK")
Message: strings with escaped "quotes" are OK
How would you translate this message into Spanish?
```

**Comment by MichaelChirico** [comment]
This was also closed in #176
